src/types/env.ts

```diff
@@ -18,13 +18,7 @@ export interface InformationNotification {
   sticky: boolean;
 }
 
-export interface SpreadsheetEnv {
-  notifyUser: (notification: InformationNotification) => any;
-  raiseError: (text: string, callback?: () => void) => any;
-  askConfirmation: (content: string, confirm: () => any, cancel?: () => any) => any;
```

Comment:
Do we really want to remove these methods from the env ? Leaving them in the SpreadsheetChildEnv doesn't cost much, and makes interacting with them a bit less annoying.

Reply:
@VincentSchippefilt  disagreed on that point and I don't have any strong opinion on this. The bad bard is that is had to become
```typescript
 notifyUser: (notification: InformationNotification) => this.notificationStore.notifyUser(notification);
```
Everything we put in the env is set in stone as we pass direct references. If we want the notifications, or the model to act as actual props, we should have (nearly) every entry onf the env being dynamic references and not static or binded as we have today.